From 48fc576ce88356a879705c139404abe4556a13ee Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Fri, 9 May 2014 13:42:27 +0800
Subject: [PATCH 1/2] ENGR00304825 Two Camera apps are not same in Launcher
 shotcut and Launcher main

Align all layout to use the Camera2 as the shotcut of Camera icon

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 res/xml-sw600dp/default_workspace.xml | 4 ++--
 res/xml-sw720dp/default_workspace.xml | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/res/xml-sw600dp/default_workspace.xml b/res/xml-sw600dp/default_workspace.xml
index 0537169..5a2dcd4 100644
--- a/res/xml-sw600dp/default_workspace.xml
+++ b/res/xml-sw600dp/default_workspace.xml
@@ -37,8 +37,8 @@
         launcher:spanX="2"
         launcher:spanY="2" />
     <favorite
-        launcher:packageName="com.android.camera"
-        launcher:className="com.android.camera.Camera"
+        launcher:packageName="com.android.camera2"
+        launcher:className="com.android.camera.CameraLauncher"
         launcher:screen="2"
         launcher:x="0"
         launcher:y="3" />
diff --git a/res/xml-sw720dp/default_workspace.xml b/res/xml-sw720dp/default_workspace.xml
index 6302d7e..185ca6d 100644
--- a/res/xml-sw720dp/default_workspace.xml
+++ b/res/xml-sw720dp/default_workspace.xml
@@ -37,8 +37,8 @@
         launcher:spanX="2"
         launcher:spanY="2" />
     <favorite
-        launcher:packageName="com.android.camera"
-        launcher:className="com.android.camera.Camera"
+        launcher:packageName="com.android.camera2"
+        launcher:className="com.android.camera.CameraLauncher"
         launcher:screen="2"
         launcher:x="0"
         launcher:y="3" />
-- 
1.8.0

